src/FluentNHibernate.Testing/Visitors/RelationshipPairingVisitorSpec.cs

```diff
@@ -0,0 +1,60 @@
+using System.Linq;
+using FakeItEasy;
+using FluentNHibernate.MappingModel;
+using FluentNHibernate.MappingModel.Collections;
+using FluentNHibernate.Visitors;
+using NUnit.Framework;
+
+namespace FluentNHibernate.Testing.Visitors
+{
+    public abstract class RelationshipPairingVisitorSpec : Specification
```

Comment:
I believe abstract class is not necessary here. If we had several test cases that can share the functionality - sure